src/Cms/Translation.php

```diff
@@ -83,6 +84,26 @@ public function direction(): string
 		return $this->get('translation.direction', 'ltr');
 	}
 
+	/**
+	 * Returns the first day of the week (0 = Sunday ... 6 = Saturday)
+	 * for the translation's locale (or as defined via config option)
+	 *
+	 * @since 4.5.0
+	 */
+	public function firstWeekday(): int
+	{
+		$weekday = App::instance()->option('date.weekday');
+
+		if (is_int($weekday) === true) {
+			return $weekday;
+		}
+
+		$locale   = $this->locale();
+		$calendar = IntlCalendar::createInstance(null, $locale);
+
+		return $calendar->getFirstDayOfWeek() - 1;
```

Comment:
What would happen if `$locale` is incorrect, invalid or not found by server? `::getFirstDayOfWeek` returns `int|false`. Are we sure that returns always int for this line?